deployment/main.py

```diff
@@ -0,0 +1,63 @@
+import argparse
+import framework
+import os
+import sys
+import yaml
+
+from deployment.deployment import deploy
+from framework.customizations.logging import logging
+
+log = logging.getLogger(__name__)
+
+
+# Directories
+TOP_DIR = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+TEMPLATE_DIR = os.path.join(TOP_DIR, "noobaa-sa-infra", "templates")
+
+
+def process_arguments(arguments):
+    """
+    This function process the arguments which are passed to noobaa-sa-infra
+
+    Args:
+        arguments (list): List of arguments
+
+    """
+    parser = argparse.ArgumentParser(add_help=False)
+    parser.add_argument("--conf", action="append", default=[])
+
+    args, unknown = parser.parse_known_args(args=arguments)
+    load_config(args.conf)
+
+
+def load_config(config_files):
+    """
+    This function load the config files in the order defined in config_files
+    list.
+
+    Args:
+        config_files (list): config file paths
+
+    """
+    for config_file in config_files:
+        with open(os.path.abspath(os.path.expanduser(config_file))) as file_stream:
+            custom_config_data = yaml.safe_load(file_stream)
+            framework.config.update(custom_config_data)
+
```

Comment:
Those function will be common for other entry points as well I think, and will be used in other entrypoint as well,so I don't think it should be under deployment folder but rather under framework package.

Reply:
moved